Package.swift

```diff
@@ -837,6 +837,18 @@ let package = Package(
         .define("FIR_VERSION", to: firebaseVersion),
       ]
     ),
+
+    // MARK: Testing support
+
+    .target(
+      name: "FirebaseFirestoreTestingSupport",
```

Comment:
Similar to the question about a single vs multiple pods, should we define a single vs. multiple Swift packages?

Reply:
I think the [response](https://github.com/firebase/firebase-ios-sdk/pull/7668#discussion_r589634538) is relevant here - I think separate targets are better.